### DISCUSSION TITLE
ubuntu compile nginx with spnego module no /config was found

### DISCUSSION THREAD

**Discussion by dbrings** [stnoonan/spnego-http-auth-nginx-module#137] [Q&A] [answered]
I've been running nginx plus subscription.
I installed nginx binary from open source nginx_1.22.1, and it runs my nginx.conf, except for the ntlm directive.
There is a module that claims to work in place of ntlm, but if the spnego-http-auth-nginx-module will work in my project, it is preferred.
I need to know if this module is compatible with nginx_1.22.1. The README.md states it is compatible with nginx 1.2 - 1.7. I'm not sure how to compare those versions with what I've got, or if I should run an older nginx version.
I am deploying on ubuntu 18.04.6 LTS.
Please help, let me know how I can succeed with this module.
Thanks!

**Comment by dbrings** [comment]
This is my first time compiling on nginx ubuntu.
I'm getting an error message: no /config was found
![2023-03-27 09_43_52-ELDERMSPWEB04 - Connected](https://user-images.githubusercontent.com/17418007/227975350-1c760a7a-6fc8-4c87-92af-4042799e5599.png)
Please help

**Comment by Alphix** [comment, marked as answer]
https://github.com/stnoonan/spnego-http-auth-nginx-module/blob/main/.github/workflows/build-test.yml contains an example of how to build the module for recent a recent version of Ubuntu.

In addition, the module is now packaged for Debian (Trixie onwards) and Ubuntu (in the universe component).
